pkgs/development/python-modules/gasp/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, tkinter, pygame }:
+
+buildPythonPackage rec {
+  pname = "gasp";
+  version = "0.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3h90H+E0pvDtpT/eQUsLjnnFholbxxk7K66exB6W3vo=";
+  };
+
+  format = "pyproject";
+
+  doCheck = false;
+
+  propagatedBuildInputs = [tkinter pygame];
+
+  meta = with lib; {
+    homepage = "https://codeberg.org/GASP/GASP_Code";
+    description = "GASP is designed to enable absolute beginners to write 1980's style arcade games as an introduction to python.";
```

```suggestion
    description = "Module which helps to write 1980's style arcade games";
```

Also, see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes